src/app/file-manager/transferables-grid/transferables-grid.component.ts

```diff
@@ -194,6 +193,13 @@ export class TransferablesGridComponent implements OnInit, AfterViewInit {
   }
 
   startDownload() {
-    this.registerDownload.startDownload(this.filesDatabase.data);
+    const files = this.filesDatabase.data.filter(item => item.type === Type.DOWNLOAD);
```

Comment:
I don't understand why the code reads from `filesDatabase` here instead of accepting `files` as an argument.